### PR TITLE
Upgrade networkx and freeze dependencies

### PR DESCRIPTION
**Pull request by maxf**
New version is out which fixes a vulnerabillity found by pen testers

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,73 @@
...
+PyJWT==1.7.1
+pymacaroons==0.13.0
+PyNaCl==1.4.0
+pyRFC3339==1.1
+python-apt===2.2.0-ubuntu0.21.04.1
+python-dateutil==2.8.1
+python-debian==0.1.39
+pytz==2021.1
+pyxdg==0.27
+PyYAML==5.3.1
```
Comment (suganya-s) [inline requirements.txt:54]: seems to be downgrading PyYaml from 5.4 to 5.3.1

File: requirements.txt [hunk 2]
```diff
@@ -1,23 +1,73 @@
-# local package
--e .
-
-# external requirements
-pyyaml==5.4
-html5lib==1.0.1
```
Comment (suganya-s) [inline requirements.txt:6]: a lot of these requirements haven't been copied over

**Review by suganya-s — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (maxf) [reply requirements.txt]: Fixed.

File: requirements.txt (hunk 2 above)
Comment (maxf) [reply requirements.txt:6]: Fixed

**Pull request closed without merging**